### PR TITLE
net-misc/unison: bump to 2.51.2

### PR DESCRIPTION
**Pull request by hsoft**
@heroxbd here it is. I had to bump `eselect-unison` as well as revbump `2.48.4` so that it slots `unison-fsmonitor`.

### CODE REVIEW

File: net-misc/unison/unison-2.51.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+IUSE="gtk doc static debug threads +ocamlopt test"
```
Comment (Zlogene) [inline net-misc/unison/unison-2.51.2.ebuild]: at least move it acording to variables order ;)

File: net-misc/unison/unison-2.51.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+# ocaml version so we are sure it has ocamlopt use flag
+DEPEND="dev-lang/ocaml[ocamlopt?]
+	gtk? ( dev-ml/lablgtk )"
+
+RDEPEND="gtk? ( dev-ml/lablgtk
+	|| ( net-misc/x11-ssh-askpass net-misc/ssh-askpass-fullscreen ) )
+	>=app-eselect/eselect-unison-0.4"
+
+SRC_URI="https://www.seas.upenn.edu/~bcpierce/unison/download/releases/${P}/${P}.tar.gz
```
Comment (Zlogene) [inline net-misc/unison/unison-2.51.2.ebuild]: this too

File: app-eselect/eselect-unison/eselect-unison-0.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+DEPEND=""
+RDEPEND=">=app-admin/eselect-1.2.3"
+
+src_install() {
+	local MODULEDIR="/usr/share/eselect/modules"
+	local MODULE="unison"
+	dodir ${MODULEDIR}
+	insinto ${MODULEDIR}
+	newins "${FILESDIR}/${MODULE}.eselect-${PVR}" ${MODULE}.eselect \
+		|| die "failed to install"
```
Comment (heroxbd) [inline app-eselect/eselect-unison/eselect-unison-0.4.ebuild]: You don't need `|| die` for ebuild helpers like `newins`.

**Comment by hsoft** [comment]
Done.

**Pull request closed without merging**